pyproject.toml

```diff
@@ -38,6 +38,9 @@ bleak-winrt = { version = "^1.2.0", markers = "platform_system=='Windows'", pyth
 "winrt-Windows.Foundation.Collections" = { version = "^2", markers = "platform_system=='Windows'", python = ">=3.12" }
 "winrt-Windows.Storage.Streams" = { version = "^2", markers = "platform_system=='Windows'", python = ">=3.12" }
 dbus-fast = { version = ">=1.83.0, < 3", markers = "platform_system == 'Linux'" }
+[project.optional-dependencies]
+bumble = ["bumble=^0.0.201"]
+
```

Comment:
Bumble does not seem to be conforming to semver.  Unless no release from 0.0.0 -> 0.0.202 broke an API or added a feature 🤣.

What this means is that users of Bumble should lock it instead of trusting its version.  Please change this to

```toml
[project.optional-dependencies]
bumble = ["bumble==0.0.201"]
```

Or 0.0.202 if I feel it is not a breaking change.

The `^=` syntax is shorthand for "update until a breaking version" - but it only works if the package uses semver.

Reply:
Good catch, 201 minor patches. Yes that makes more sense